src/encoded/static/components/matrix_mouse_development.js

```diff
@@ -155,6 +155,43 @@ function sortMouseArray(a, b) {
     return aNumerical - bNumerical;
 }
 
+// Generate object listing all ages corresponding to a mouse stage based on rowCategoryData
+const getMouseAgeObject = (rowCategoryData) => {
+    const mouseAgeFullArray = [];
+    rowCategoryData.forEach((datum) => {
```

Comment:
This is the first time I have ever seen a developer use datum for a single part of data.